elements/neon.md/neon.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+title: Neon
+---
+
+Symbol : Ne
+Atomic Number : 10
+
```

Comment:
Did they use the right page format?
 Yes
Is their file name correct?
 Yes
Did they add information about the element?
 Yes